src/main/metabase/examples.ts

```diff
@@ -142,8 +143,13 @@ function textGeneralizedSQL(field: Field, table: Table, averageLength: number):
   const bucket = `substring(${postgresQuote(column)}, 1, ${nChars})`;
 
   return tableMiniBar({
-    name: `${table.display_name} by ${field.display_name}`,
-    sql: lines(`SELECT ${bucket} || ${stars}, count(*)`, `FROM ${postgresQuote(table.name)}`, `GROUP BY ${bucket}`),
+    name: `${table.display_name} by ${field.display_name} ${nChars} initial characters`,
+    sql: lines(
+      `SELECT ${bucket} || ${stars} as ${postgresQuote(field.name + '_starts_with')}, count(*)`,
```

Comment:
I would keep the `field.name`:
```suggestion
      `SELECT ${bucket} || ${stars} as ${postgresQuote(field.name)}, count(*)`,
```
The censoring will be obvious from the asterisks.

Reply:
but this leads to the wrong conclusion, that we're only obfuscating actual entries, i.e. `Sm***` actually stands for `Smith` not a whole bunch of distinct values. I'd even be inclined to drop the asterisks in favor of this `_starts_with`

Reply:
Alright, I'm ok with current name.